spec/integration/guest_view_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe 'the guest view', type: :feature do
+  #include Capybara::DSL
+
+  describe 'the home page' do
+    xit "has a navbar" do
+      visit root_url
+      within("#nav-bar") do
+        expect(page).to have_link("Main")
+        expect(page).to have_link("Menu")
+        expect(page).to have_link("Login")
+      end
+    end
+
+    xit "goes to the menu page" do
+      page.click_link("Menu")
+      expect(current_path).to eq(menu_path)
+    end
+  end
+
+  describe 'the menu view' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.